exercises/practice/Practice.sln

```diff
@@ -0,0 +1,136 @@
+﻿
```

Comment:
I added the solution as it makes testing a _lot_ faster than individually running dotnet test on each exercise.